docker/docker-compose-ngrok.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.3'
+# See "Docker ngrok" section in docker/README
+
+services:
+  ngrok:
+    container_name: jetpack_ngrok
+    ports:
+      - "0.0.0.0:4040:4040"
+    image: wernight/ngrok
+    links:
+      - "wordpress"
+    env_file:
+      - default.env
+      - .env
+    environment:
+      - NGROK_PORT=wordpress:80
```

Comment:
Will this port `80` work if user runs their instance in custom port as described in docs: https://github.com/Automattic/jetpack/blob/64dcdac25617ef5626f90885ba744cb318777604/docker/README.md#host-environment

You might just need to use the same env variable like we use in main docker-compose file:

https://github.com/Automattic/jetpack/blob/64dcdac25617ef5626f90885ba744cb318777604/docker/docker-compose.yml#L79-L80

Reply:
No, this is correct as it is :)
`wordpress` container always exposes port 80 which can be redirected to configured port on your computer with 80 as default as well with `"${PORT_WORDPRESS:-80}:80"`. But on container, its hardcoded 80.
My ngrok container connects to port 80 on the `wordpress`, **not your computer** . That means, that technically you don't need to expose **ANY**  port on your computer to the wordpress container and it will still work through the tunnel.